infra/database.tf

```diff
@@ -123,7 +125,7 @@ resource "aws_security_group" "database" {
     from_port       = 5432
     to_port         = 5432
     protocol        = "tcp"
-    security_groups = [aws_security_group.api.id, aws_security_group.app.id]
+    security_groups = [module.vpc.default_security_group_id]
```

Comment:
This ingress block can be removed. The self ingress block alone should be sufficient. We can simply add this security_group_id as one of the service security groups in the ECS service.